pom.xml

```diff
@@ -115,4 +115,7 @@
         </repository>
     </repositories>
 
+    <properties>
+        <maven.findbugsplugin.version>3.0.5</maven.findbugsplugin.version>
```

Comment:
This is needed as the version coming from parent (3.0.3) is not compatible with later maven versions.